content/home.ts

```diff
@@ -118,17 +118,17 @@ const content: HomeContent = {
     {
       id: 'faq4',
       summary: 'If my conviction meets the eligibility criteria, does that guarantee that I can vacate it?',
-      details: 'A request to vacate may be denied for other reasons, and is up to the discretion of the judge. The most common reasons that a request can be denied is if:\n1. the court does not believe that vacating your conviction will be in the interest of society,\n2. you violated probation or have not paid off your fines,\n3. you are otherwise not eligible to vacate that particular conviction, or\n4. there was an inaccuracy in the court records and/or the application.',
+      details: 'A request to vacate may be denied for other reasons, and is up to the discretion of the judge. The most common reasons that a request can be denied is if:\n <ol><li> the court does not believe that vacating your conviction will be in the interest of society,</li><li> you violated probation or have not paid off your fines,</li><li> you are otherwise not eligible to vacate that particular conviction, or</li><li> there was an inaccuracy in the court records and/or the application.</li></ol>',
```

Comment:
I think that the ol and li tags here are causing one of the warnings. Is there a reason why the previous version was not working?

Reply:
Without <ol>, the items' numbers font would have remained same as before instead of becoming body2. It's fixed now. in the new commit.

Reply:
It seems that other errors are old and exist on staging too.